src/modules/class-wpml-elementor-multiple_gallery.php

```diff
@@ -0,0 +1,35 @@
+<?php
+/**
+ * WPML_Elementor_Multiple_Gallery
+ */
+namespace WPML\PB\Elementor\Modules;
+
+class WPML_Elementor_Multiple_Gallery extends \WPML_Elementor_Module_With_Items {
+
+    protected function get_title($field)
+    {
```

Comment:
@GeorgeBotsev, please review the formatting, it should be:

```php
protected function get_title( $field ) {
```

Reply:
Adjusted the formatting.